src/sqlfluff/dialects/dialect_postgres_keywords.py

```diff
@@ -925,18 +925,21 @@ def get_keywords(keyword_list, keyword_type):
     ("DATE", "non-reserved"),
     ("DEPTH", "non-reserved"),
     ("DESCRIBE", "non-reserved"),
+    ("DETERMINISTIC", "non-reserved"),
```

Comment:
If someone can explain what goes into `postgres_docs_keywords` vs `postgres_nondocs_keywords`, it would be great to document that in the code. I just guessed.

Reply:
@WittierDinosaur ?

I'm guessing from the name that `postgres_nondocs_keywords` refers to keywords that appear to be keywords but are not listed in https://www.postgresql.org/docs/current/sql-keywords-appendix.html

Reply:
Does the distinction even matter? Does it matter if I've put in the "wrong" place for now?

Reply:
No it doesn’t presently. Type is important but not which list it appears in.